core/jvm/src/main/scala/fs2/rx/package.scala

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2013 Functional Streams for Scala
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package fs2
+
+import cats.effect.kernel._
+import java.util.concurrent.Flow._
+
+/** Implementation of the reactive-streams protocol for fs2; based on Java Flow.
+  *
+  * @example {{{
+  * scala> import fs2._
+  * scala> import fs2.rx.{fromPublisher, toPublisher}
+  * scala> import cats.effect.{IO, Resource}, cats.effect.unsafe.implicits.global
+  * scala>
+  * scala> val upstream: Stream[IO, Int] = Stream(1, 2, 3).covary[IO]
+  * scala> val publisher: Resource[IO, Publisher[Int]] = toPublisher(upstream)
+  * scala> val downstream: Stream[IO, Int] = Stream.resource(publisher).flatMap(p => fromPublisher(p, bufferSize = 16))
+  * scala>
+  * scala> downstream.compile.toVector.unsafeRunSync()
+  * res0: Vector[Int] = Vector(1, 2, 3)
+  * }}}
+  *
+  * @see [[http://www.reactive-streams.org/]]
+  */
+package object rx {
```

Comment:
Following the rest of FS2, the package should be `interop.flow` maybe?

https://docs.oracle.com/javase/9/docs/api/java/util/concurrent/Flow.html

Reply:
I was about to ask what should be the right package name.
`interop.flow` sounds good.